lib/autoyast.rb

```diff
@@ -266,6 +266,12 @@ def apply_extracted_files(scope)
     return if !@system_description[scope] || !@system_description[scope].extracted
 
     base = Pathname(@system_description.scope_file_store(scope).path)
+
+    @system_description[scope].files.select { |file| file.link? }.each do |link|
+      @chroot_scripts << "rm -rf '#{link.name}'"
+      @chroot_scripts << "ln -s '#{link.target}' '#{link.name}'"
+    end
```

Comment:
Is this code used at all? This is only called for the config-files scope, where we haven't done the changes to extract the necessary informatioin, isn't it?